homeassistant/components/sensor/forecast.py

```diff
@@ -24,6 +24,7 @@
     - pressure
     - visibility
     - ozone
+  units: si, us, ca, uk, uk2 or auto(default)
```

Comment:
uk has been [deprecated](https://developer.forecast.io/docs/v2#options) and should not be used. Let's not suggest it.